<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BracersFilterDialog</class>
 <widget class="QDialog" name="BracersFilterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>281</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Номер браслета: </string>
       </property>
       <property name="buddy">
        <cstring>bracer_number_spin</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="bracer_number_spin">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_11">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Тариф: </string>
       </property>
       <property name="buddy">
        <cstring>tariff_comboBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="tariff_comboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="styleSheet">
      <string notr="true">color: rgb(100, 100, 100);</string>
     </property>
     <property name="title">
      <string>Дополнительно</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0,1,0,1">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 0);</string>
          </property>
          <property name="text">
           <string>Вход №:          </string>
          </property>
          <property name="buddy">
           <cstring>enter_number_spin</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="enter_number_spin">
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 0);</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 0);</string>
          </property>
          <property name="text">
           <string>Выход №:</string>
          </property>
          <property name="buddy">
           <cstring>exit_num_spin</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="exit_num_spin">
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 0);</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="0,1">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 0);</string>
          </property>
          <property name="text">
           <string>Кол-во детей:  </string>
          </property>
          <property name="buddy">
           <cstring>childs_spin</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="childs_spin">
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 0);</string>
          </property>
          <property name="maximum">
           <number>9999</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">color: rgb(100, 100, 100);</string>
     </property>
     <property name="title">
      <string>Время входа</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 0);</string>
          </property>
          <property name="text">
           <string>От: </string>
          </property>
          <property name="buddy">
           <cstring>in_from_dateTime</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateTimeEdit" name="in_from_dateTime">
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 0);</string>
          </property>
          <property name="dateTime">
           <datetime>
            <hour>0</hour>
            <minute>0</minute>
            <second>0</second>
            <year>2018</year>
            <month>1</month>
            <day>1</day>
           </datetime>
          </property>
          <property name="date">
           <date>
            <year>2018</year>
            <month>1</month>
            <day>1</day>
           </date>
          </property>
          <property name="minimumDateTime">
           <datetime>
            <hour>0</hour>
            <minute>0</minute>
            <second>0</second>
            <year>1818</year>
            <month>1</month>
            <day>1</day>
           </datetime>
          </property>
          <property name="currentSection">
           <enum>QDateTimeEdit::NoSection</enum>
          </property>
          <property name="displayFormat">
           <string>Дата: dd | MM | yyyy   Время: H:mm</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 0);</string>
          </property>
          <property name="text">
           <string>До: </string>
          </property>
          <property name="buddy">
           <cstring>in_to_dateTime</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateTimeEdit" name="in_to_dateTime">
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 0);</string>
          </property>
          <property name="dateTime">
           <datetime>
            <hour>0</hour>
            <minute>0</minute>
            <second>0</second>
            <year>2018</year>
            <month>1</month>
            <day>1</day>
           </datetime>
          </property>
          <property name="date">
           <date>
            <year>2018</year>
            <month>1</month>
            <day>1</day>
           </date>
          </property>
          <property name="minimumDateTime">
           <datetime>
            <hour>0</hour>
            <minute>0</minute>
            <second>0</second>
            <year>1818</year>
            <month>1</month>
            <day>1</day>
           </datetime>
          </property>
          <property name="currentSection">
           <enum>QDateTimeEdit::NoSection</enum>
          </property>
          <property name="displayFormat">
           <string>Дата: dd | MM | yyyy   Время: H:mm</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="today_for_enter_but">
        <property name="text">
         <string>Сегодня</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="exit_group">
     <property name="styleSheet">
      <string notr="true">color: rgb(100, 100, 100);</string>
     </property>
     <property name="title">
      <string>Время выхода</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 0);</string>
          </property>
          <property name="text">
           <string>От: </string>
          </property>
          <property name="buddy">
           <cstring>out_from_dateTime</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateTimeEdit" name="out_from_dateTime">
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 0);</string>
          </property>
          <property name="dateTime">
           <datetime>
            <hour>0</hour>
            <minute>0</minute>
            <second>0</second>
            <year>2018</year>
            <month>1</month>
            <day>1</day>
           </datetime>
          </property>
          <property name="date">
           <date>
            <year>2018</year>
            <month>1</month>
            <day>1</day>
           </date>
          </property>
          <property name="minimumDateTime">
           <datetime>
            <hour>0</hour>
            <minute>0</minute>
            <second>0</second>
            <year>1818</year>
            <month>1</month>
            <day>1</day>
           </datetime>
          </property>
          <property name="currentSection">
           <enum>QDateTimeEdit::NoSection</enum>
          </property>
          <property name="displayFormat">
           <string>Дата: dd | MM | yyyy   Время: H:mm</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,1">
        <item>
         <widget class="QLabel" name="label_13">
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 0);</string>
          </property>
          <property name="text">
           <string>До: </string>
          </property>
          <property name="buddy">
           <cstring>out_to_dateTime</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateTimeEdit" name="out_to_dateTime">
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 0);</string>
          </property>
          <property name="dateTime">
           <datetime>
            <hour>0</hour>
            <minute>0</minute>
            <second>0</second>
            <year>2018</year>
            <month>1</month>
            <day>1</day>
           </datetime>
          </property>
          <property name="date">
           <date>
            <year>2018</year>
            <month>1</month>
            <day>1</day>
           </date>
          </property>
          <property name="minimumDateTime">
           <datetime>
            <hour>0</hour>
            <minute>0</minute>
            <second>0</second>
            <year>1818</year>
            <month>1</month>
            <day>1</day>
           </datetime>
          </property>
          <property name="currentSection">
           <enum>QDateTimeEdit::NoSection</enum>
          </property>
          <property name="displayFormat">
           <string>Дата: dd | MM | yyyy   Время: H:mm</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="today_for_exit_but">
        <property name="text">
         <string>Сегодня</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BracersFilterDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>429</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BracersFilterDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>297</x>
     <y>435</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
